export.py

```diff
@@ -0,0 +1,85 @@
+
+import yaml
+import csv
+import json
+import csv
+import os
+def read_yaml(file_path):
+    with open(file_path, 'r') as file:
+        data = yaml.safe_load(file)
+    return data
+
+def clean_value(value):
+    if isinstance(value, list):
+        return ', '.join(map(str, value))
```

Comment:
The format of dictionaries in the CSV was weird, so I figure JSON is a good format, and it works well for both lists and dicts:
```suggestion
    if isinstance(value, list) or isinstance(value, dict):
        return json.dumps(value)
```

Reply:
@idris 

```
   if isinstance(value, list):
        return ', '.join(map(str, value))
```
    
brands.csv:
    
<img width="725" alt="Screenshot 2024-01-18 at 7 15 18 PM" src="https://github.com/TechForPalestine/boycott-israeli-consumer-goods-dataset/assets/79056754/56d97b86-6a2c-4334-8700-4a1dfa0570ed">


```
   if isinstance(value, list) or isinstance(value, dict):
        return json.dumps(value)
```

 brands.csv:

<img width="772" alt="Screenshot 2024-01-18 at 7 19 48 PM" src="https://github.com/TechForPalestine/boycott-israeli-consumer-goods-dataset/assets/79056754/5bb10c6c-0a7a-439a-a411-c18d2945bf82">


I used `', '.join(map(str, value))` to remove `[]` and `""` in the csv files.
and I think dict format does not exist here because `isinstance(value, dict)` is always `false`

Reply:
Hm, I think you're right that the `', '.join(...)` version is better for lists.

If you look at the sabra row, there is a dict for the `stakeholders` column, which shows up like this:
`"{'id': 'pepsico', 'type': 'owner', 'percent': 50}, {'id': 'strauss-group', 'type': 'owner', 'percent': 50}"`

We're not really sure how the `stakeholders` bit will pan out, so I'm just going to leave your join as-is, and remove `stakeholders` from the CSV export for now.

Reply:
oooh the plot thickens.. it's an array of objects.  Even more reason to ignore that for now.